iv.yaml

```diff
@@ -1,7 +1,8 @@
 lenguaje: python
 entidad:
-  - src/respuesta.py
-  - src/resolucionExamen.py
+  - supercatch/tokensRespuesta.py
 automatizar:
   fichero: pyproject.toml
   orden: poe
+test:
+  - tests/test_pnl.py
```

Comment:
... y menos de forma inconsistente. O usas CamelCase o usas snake_case, pero no las dos.

Reply:
El hecho de usar esta forma concreta para el fichero de test lo realicé atendiendo a  [Pytest: Convenions for test discovery](https://docs.pytest.org/en/7.2.x/explanation/goodpractices.html#conventions-for-python-test-discovery). Tenía pensado utilizar esta característica en un futuro al igual que  he hecho con la ejecución paralela de los tests, pero por el momento no he estudiado la integración de esta utilidad en pytest.